src/app/components/logout-button/logout-button.component.ts

```diff
@@ -35,9 +37,7 @@ class LogoutButtonController {
 
     simpleWalletProvider.clean();
 
-    this.$state.go('vicigo.feeds');
-
-    location.reload();
+    window.location.href = `${clientURL}/login`;
```

Comment:
Why an absolute path? Relative will not work?

Reply:
in case there is some kind of a situation where you prefer your clientURL to be something else. the other urls that we have has the same structure.
relative would work aswell if you prefer that

Reply:
As long as its not coming, I think it's ok to have relative one.

Reply:
changed it to relative path